scripts/test_node.sh

```diff
@@ -73,7 +73,9 @@ from_scratch () {
   update_test_genesis '.app_state["gov"]["deposit_params"]["min_deposit"]=[{"denom": "ujuno","amount": "1000000"}]'
   update_test_genesis '.app_state["crisis"]["constant_fee"]={"denom": "ujuno","amount": "1000"}'  
 
-  update_test_genesis '.app_state["tokenfactory"]["params"]["denom_creation_fee"]=[{"denom":"ujuno","amount":"100"}]'
+  # update_test_genesis '.app_state["tokenfactory"]["params"]["denom_creation_fee"]=[{"denom":"ujuno","amount":"100"}]'
+  update_test_genesis '.app_state["tokenfactory"]["params"]["denom_creation_fee"]=[]'
+  update_test_genesis '.app_state["tokenfactory"]["params"]["denom_creation_gas_consume"]=2000000'
```

Comment:
basically, we will make this a high gas tx, instead of a fee'd tx?

Reply:
exactly

This passes the cost on to the user executing, rather than the contract developer directly (if a new denom is needed to be created)

Current mainnet is still 1JUNO cost until larry puts up a para change proposal - [https://commonwealth.im/juno/discussion/10596-parameter-change-reduce-denom_creation_fee-to-zero](https://commonwealth.im/juno/discussion/10596-parameter-change-reduce-denom_creation_fee-to-zero)